lib/SyTest/Homeserver/Dendrite.pm

```diff
@@ -195,37 +195,41 @@ sub _get_config
 
       mscs => {
          database => {
-            connection_string => 
+            connection_string =>
                ( ! defined $ENV{'POSTGRES'} || $ENV{'POSTGRES'} == '0') ?
                "file:$self->{hs_dir}/mscs.db" : $db_uri,
          },
          mscs => ["msc2836", "msc2946", "msc2444", "msc2753"],
       },
 
+      push_server => {
```

Comment:
@tommie I just realized that we already have `push_server` section below

Reply:
I've fixed that. Good catch.

Reply:
Sorry about that. Thank you!